.github/workflows/tests.yml

```diff
@@ -287,6 +289,10 @@ jobs:
             features: ["rdrand"]
           - target: x86_64-unknown-l4re-uclibc
             features: ["rdrand"]
+          # `std` support still in progress. Can be moved up with the other
+          # apple targets after https://github.com/rust-lang/rust/pull/103503
+          - target: aarch64-apple-tvos
```

Comment:
Nit: how this rule works is a little confusing, but you don't need to have it under `include:`. No features is the default for the targets in the matrix.

If you move this comment to above where you have `aarch64-apple-tvos` in the target list, that should be fine.